CHANGELOG.md

```diff
@@ -72,6 +72,7 @@
 * [BUGFIX] Ruler Experimental API: Don't allow rule groups without names or empty rule groups. #3210
 * [BUGFIX] Experimental Alertmanager API: Do not allow empty Alertmanager configurations or bad template filenames to be submitted through the configuration API. #3185
 * [BUGFIX] When using ruler sharding, moving all user rule groups from ruler to a different one and then back could end up with some user groups not being evaluated at all. #3235
+* [BUGFIX] Clarify 'implements ChunkStore' doc comments. #3085
```

Comment:
Doesn't belong in release notes as it has no impact on users.

Reply:
Fair enough! Gone.